app/models/work.rb

```diff
@@ -0,0 +1,24 @@
+class Work < ApplicationRecord
+  belongs_to :category
+
```

Comment:
It's interesting to me that you made `category` a separate table. The other option would be to make it a string field on `work` and restrict the value with a validation (see the instructor MediaRanker for an example of this). This technique certainly solves the problem, and it provides some interesting flexibility, but I think it might be more work than is needed for this particular application.